x/oracle/keeper/feedermanagement/feedermanager.go

```
@@ -0,0 +1,921 @@
+	// commit state of mature rounds
+	f.updateAndCommitRounds(ctx)
+
+	// set status to open for rounds before their quoting window
+	feederIDs := f.prepareRounds(ctx)
+	// remove nocnes for closing quoting-window and set nonces for opening quoting-window
+	f.setupNonces(ctx, feederIDs)
+
+	f.ResetFlags()
+
+	f.updateCheckTx()
+
```

golangci-lint (1.62.2)

File is not `gofumpt`-ed

(gofumpt)